src/Symfony/Component/Validator/Validator/PropertyCandidate.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Validator;
+
+/**
+ * A wrapper for a callable initializing a property from a getter.
+ *
+ * @internal
+ */
+class PropertyCandidate
```

Comment:
I have trouble naming that class and choosing the right namespace though.

Reply:
What about something like `LazyValue`/`LazyPropertyValue`? That would be inline with other places in the framework where we also delay resolving a value.

Reply:
Renamed 👍